### ISSUE TITLE
Move user/aircraft to diversion airport

### ISSUE THREAD

**Issue by nabeelio** [nabeelio/phpvms#1309]
- Add new column `diversion_airport_id`
- Populate it from ACARS
- On accept, move user/aircraft to that location
- If airport doesn't exist, try to do a lookup from API

**Comment by nabeelio** [comment]
@FatihKoz I can't remember, is this considered done?

**Comment by FatihKoz** [comment]
Nope, I solved it with a 3rd party module of my own. We talked about this
and you wanted me to implement my solution to core but I never had enough
time to work on that.

Logic was simple;

Check diversion record from pirep-fields when it gets filed
Check the airport and try to add the airport to database if it is not there
> if successful or airport already exists, move assets to this new airport
and remove flight_id from that pirep
> if not successful, do nothings (scenery problems) record only diversion
note
> try to determine the reason of diversion (scenery error, crash,
operational etc)

I still can do this as I do have a working logic but still have zero free
time :(
